runtime/lua/vim/shared.lua

```diff
@@ -107,13 +107,13 @@ end
 ---
 ---@see |vim.gsplit()|
 ---
----@param s String to split
----@param sep Separator string or pattern
----@param kwargs Keyword arguments:
+---@param s string String to split
+---@param sep string Separator or pattern
+---@param kwargs table|boolean Keyword arguments:
```

Comment:
`kwargs` can technically be a Boolean but that’s only for backward compatibility. We should only advertise it as a table argument.